akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1452,6 +1452,20 @@ trait FlowOps[+Out, +Mat] {
   def idleTimeout(timeout: FiniteDuration): Repr[Out] = via(new Timers.Idle[Out](timeout))
 
   /**
+    * If the time between emission of element and following backpressure notification exceed the provided timeout,
```

Comment:
We are always careful with adding too many stages. Depends on how useful it is and if the alternative "cookbook recipie" is good enough.